web/src/engine/element-wrappers/src/contentEditable.ts

```diff
@@ -103,12 +103,12 @@ export default class ContentEditable extends OutputTarget<{}> {
   }
 
   getDeadkeyCaret(): number {
-    return (this.getTextBeforeCaret() ?? this.getText()).kmwLength();
+    return this.getTextBeforeCaret().kmwLength();
```

Comment:
This is missing the fix for getTextBeforeCaret

Reply:
See below, in `getTextBeforeCaret` - the fix is implemented there.

Reply:
Curious, when I added that comment, the fix was definitely not visible in GH -- I had to expand the source to find `getTextBeforeCaret()`. Very weird. LGTM now.